src/Data/Nat/Base.agda

```diff
@@ -129,6 +129,15 @@ m ≥′ n = n ≤′ m
 _>′_ : Rel ℕ 0ℓ
 m >′ n = n <′ m
 
+------------------------------------------------------------------------
+-- Similar to _≤′_
+
+infix 4 _≤‴_
+
+data _≤‴_ : ℕ → ℕ → Set where
```

Comment:
Can this be moved below `_≤''_` so they're in the right order numerically.

Reply:
Even so, I think they should still go in order. It'd be good if you could expand the comment as well to explain that it's useful for induction.